apps/pie-docs/src/components/modal/overview/sizes.json

```diff
@@ -0,0 +1,39 @@
+{
+  "headings": [
+    "Size",
+    "Breakpoint",
+    "Width and height"
```

Comment:
Could we have separate columns for width, height, max width and max height?

Reply:
@nazanin

Reply:
Yeah that's a good point, I'll take a look at this on Monday